components/date-time/index.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { DatePicker } from './date';
+import { TimePicker } from './time';
+
+export { DatePicker } from './date';
```

Comment:
Since we already imported those, we could drop the `from`

`export { DatePicker, TimePicker }`